pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -195,13 +195,17 @@ type ActualStateOfWorldMounterUpdater interface {
 	MarkDeviceAsUnmounted(volumeName v1.UniqueVolumeName) error
 
 	// Marks the specified volume's file system resize request is finished.
-	MarkVolumeAsResized(podName volumetypes.UniquePodName, volumeName v1.UniqueVolumeName) error
+	MarkVolumeAsResized(volumeName v1.UniqueVolumeName, claimSize resource.Quantity) bool
 
 	// GetDeviceMountState returns mount state of the device in global path
 	GetDeviceMountState(volumeName v1.UniqueVolumeName) DeviceMountState
 
 	// GetVolumeMountState returns mount state of the volume for the Pod
 	GetVolumeMountState(volumName v1.UniqueVolumeName, podName volumetypes.UniquePodName) VolumeMountState
+
+	// MarkForInUseExpansionError marks the volume to have in-use error during expansion.
+	// volume expansion must not be retried for this volume
+	MarkForInUseExpansionError(volumeName v1.UniqueVolumeName)
```

Comment:
Move this to `type ActualStateOfWorld interface` in `kubernetes/kubernetes/pkg/kubelet/volumemanager/cache/actual_state_of_world.go`. The methods in `ActualStateOfWorldAttacherUpdater` are common for both A/D controller and kubelet volume manager. Since this only applies to the later we don't need to put it in the common interface.

Reply:
same as above.